### DISCUSSION TITLE
用户自己添加的菜单About点击后字体没有变粗

### DISCUSSION THREAD

**Discussion by ghost** [HEIGE-PCloud/DoIt#315] [Q&A] [answered]
我自己添加的菜单About点击后字体没有变粗，与 Posts、Tags、Categories 不统一。
请问是哪里的问题？

如图：
<img width="282" alt="Screenshot 2021-10-30 at 14 03 21" src="https://user-images.githubusercontent.com/29486126/139524235-5d802976-3d30-4816-a824-8c63baad9563.png">

<img width="282" alt="Screenshot 2021-10-30 at 14 03 29" src="https://user-images.githubusercontent.com/29486126/139524242-3e2833d0-57e4-4bdb-8fd1-3e802121e319.png">

<img width="282" alt="Screenshot 2021-10-30 at 14 03 35" src="https://user-images.githubusercontent.com/29486126/139524253-f95227e4-18a5-477d-b428-30dca83d2c49.png">

<img width="282" alt="Screenshot 2021-10-30 at 14 03 41" src="https://user-images.githubusercontent.com/29486126/139524261-4c7f0ddd-4afd-4267-9d41-2ce57c85d773.png">

**Comment by HEIGE-PCloud** [comment]
你的浏览器缓存没有刷新。

**Comment by ghost** [comment]
> 你的浏览器缓存没有刷新。

应该不是缓存的问题。
我清空缓存，或更换浏览器，或在手机端访问，都不是粗体字。

请问我该如何解决?

**Comment by ghost** [comment, marked as answer]
新建菜单时，需要将 md 文件置于文件夹中，如：/about/index.md
